### PR TITLE
update frr readme

### PR DESCRIPTION
**Pull request by samccann**
##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

documentation
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+
+    ansible-galaxy collection install frr.frr
+
+You can also include it in a `requirements.yml` file and install it with `ansible-galaxy collection install -r requirements.yml`, using the format:
+
+```yaml
+---
+collections:
+  - name: frr.frr
+    version: 0.0.1
```
Comment (NilashishC) [inline README.md]:
```suggestion
    version: 1.0.0
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+    ansible-galaxy collection install frr.frr
+
+You can also include it in a `requirements.yml` file and install it with `ansible-galaxy collection install -r requirements.yml`, using the format:
+
+```yaml
+---
+collections:
+  - name: frr.frr
+    version: 1.0.0
```
Comment (samccann) [inline README.md]:
```suggestion
```

**Comment by samccann** [comment]
Ends up the version isn't necessary for requirements.txt so gonna start removing them from the readme files so they don't get out of date.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
+
+
+# Free Range Routing (FRR) Collection
+[![CI](https://zuul-ci.org/gated.svg)](https://dashboard.zuul.ansible.com/t/ansible/project/github.com/ansible-collections/frr.frr) <!--[![Codecov](https://img.shields.io/codecov/c/github/ansible-collections/vyos)](https://codecov.io/gh/ansible-collections/frr.frr)-->
```
Comment (NilashishC) [inline README.md:4]: AFAIK, we do not have CI in place for FRR.

Comment (NilashishC) [reply README.md:4]: Well, this won't be the case in some time. :)

**Review by NilashishC — APPROVED** [review]

**Pull request merged**